manifests/m/Microsoft/AzureDataStudio/1.48.1/Microsoft.AzureDataStudio.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.AzureDataStudio
+PackageVersion: 1.48.1
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /mergetasks=!runcode
+UpgradeBehavior: install
+Commands:
+- azuredatastudio
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://sqlopsbuilds.azureedge.net/stable/8825ec21e67a34ede00e4f02a4afc692eb6cab61/azuredatastudio-windows-user-setup-1.48.1.exe
+  InstallerSha256: DDF604B35A25A6C7FDC37795DA28A6481DC7BAE50CF25610F680C474724FE8A3
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://sqlopsbuilds.azureedge.net/stable/8825ec21e67a34ede00e4f02a4afc692eb6cab61/azuredatastudio-windows-setup-1.48.1.exe

```

Comment:
These urls will be decommissioned at the end of the month. the user installer should be https://download.microsoft.com/download/d/6/f/d6f2673f-5240-4605-8e7d-5b6c49d188e8/azuredatastudio-windows-user-setup-1.48.1.exe and the machine installer should be https://download.microsoft.com/download/d/6/f/d6f2673f-5240-4605-8e7d-5b6c49d188e8/azuredatastudio-windows-setup-1.48.1.exe